ui/sign-in.reel/sign-in.html

```diff
@@ -71,13 +71,25 @@
                 "element": {"#": "submit"},
                 "value": "Sign In"
             }
+        },
+
+        "browserMessage": {
+            "prototype": "montage/ui/text.reel",
+            "properties": {
+                "element": {"#": "browserMessage"},
+                "value": "We're sorry but at this time FreeNAS only supports the Chrome browser."
+            },
+            "bindings": {
+                "classList.has('hide')": {"<-": "@owner.isBrowserSupported"}
```

Comment:
You're right. I would rather keep everything consistent. I just need to wrap my head around the idea that every `is-*` is always a true statement in CSS land. I'm fine with using `is-hidden` `is-shown` `has-options`  etc... It's also a way a good way to visually see any css classname that is being toggled by js.